charts/athens-proxy/ci/basic-values.yaml

```diff
@@ -6,3 +6,7 @@ ingress:
       paths:
         - path: /
           pathType: ImplementationSpecific
+jaeger:
+  enabled: true
+extraLabels:
+  athensIs: "awesome"
```

Comment:
I added these after reading the contributing guide in order to exercise more code paths in CI